backend/views/syson-diagram-common-view/src/main/java/org/eclipse/syson/diagram/common/view/services/ViewEdgeToolService.java

```diff
@@ -297,9 +297,26 @@ public EdgeTool createFeatureTypingEdgeTool(List<NodeDescription> targetElementD
                 .variableName("newInstance")
                 .children(changeContextNewInstance.build());
 
+        var feedback = this.viewBuilderHelper.newChangeContext()
+                .expression("aql:self.infoMessage('A feature typing already exists between these elements.')")
```

Comment:
The actual wording of the message is of course open for discussion.

Reply:
Please use `ServiceMethod` to call the AQL expression.